### PR TITLE
Ds 3945 simulator days instances

### PR DESCRIPTION
**Pull request by hcardoso81**
image

### CODE REVIEW

**Review by ornella-688 — APPROVED** [review]
good!

File: admin/js/script.js [hunk 1]
```diff
@@ -7,25 +7,83 @@ document.addEventListener('DOMContentLoaded', () => {
...
+    checkRadiosSimulatorDuringDays();
 
     function clickRadiosPhase(phase) {
         document.getElementById('cardCurrentDay').classList.add('d-none');
         if (phase === "during")
             document.getElementById('cardCurrentDay').classList.remove('d-none');
     }
 
+    function clickRadiosPhaseSimulator(phase) {
+        console.log(phase);
```
Comment (MatiasRoy-Rodriguez) [inline admin/js/script.js]: Se escapo un console

**Review by MatiasRoy-Rodriguez — APPROVED** [review]
Good

File: admin/js/script.js (hunk 1 above)
Comment (hcardoso81) [reply admin/js/script.js]: done!

**Pull request merged**